mantle/cosa/cosa_v1.go

```diff
@@ -74,6 +74,7 @@ type BuildArtifacts struct {
 	LiveIso       *Artifact `json:"live-iso,omitempty"`
 	LiveKernel    *Artifact `json:"live-kernel,omitempty"`
 	Metal         *Artifact `json:"metal,omitempty"`
+	Metal4KNative *Artifact `json:"metal4k,omitempty"`
```

Comment:
Did this just slip in from a different commit?

Reply:
Looks like it came from 078bf719

Reply:
Rebased to pick up that change.